app/controllers/contestants_controller.rb

```diff
@@ -2,7 +2,7 @@ class ContestantsController < ApplicationController
   before_action :require_signin
 
   def index
-    @contestants = Contestant.first(5) # TODO: all contestants of current round
+    @contestants = Contestant.all # TODO: all contestants of current round
```

Comment:
The # TODO comment is not necessary anymore, right? Should be removed?

Reply:
I want to make many different rounds possible. There is only one round for this app right now. But, I haven't thought of how yet 😄 I'll remove these comments anyway because they don't have much meaning (and we have #9 issue)